<dec f='include/c++/4.8.2/bits/functexcept.h' l='97' type='void std::__throw_future_error(int )'/>
<use f='include/c++/4.8.2/future' l='362' u='c' c='_ZNSt13__future_base11_State_base13_M_set_resultESt8functionIFSt10unique_ptrINS_12_Result_baseENS3_8_DeleterEEvEEb'/>
<use f='include/c++/4.8.2/future' l='385' u='c' c='_ZNSt13__future_base11_State_base21_M_set_retrieved_flagEv'/>
<use f='include/c++/4.8.2/future' l='464' u='c' c='_ZNSt13__future_base11_State_base8_S_checkERKSt10shared_ptrIT_E'/>
